### ISSUE TITLE
Confusion

### ISSUE THREAD

**Issue by devcoinfet** [rfdslabs/Mimosa-Framework#4]
is this how you would use this?
add_target 127.0.0.1 ciscoadmin ciscoadmin ftp://admin:1234@192.168.1.1/mimosa_logs

that's obviously all fake info for security reasons is that correct usage?

**Comment by devcoinfet** [comment]
Small update after review of the code it seems pexpect is setup for Tacas Mode I suggest an update to allow for use of non tacas routers

**Comment by devcoinfet** [comment]
I have almost completed my new version of mimosa that supports new options and also allows for specification of a capture point name etc.

I have made it so it works with telnet username password auth

Also  ssh based auth 
I should have a working copy ready for your review soon. 
As well I have modified mimosa cli and done a few additions to it are You accepting pulls?

**Comment by rfdslabs** [comment]
Hey devcoinfet,

Thanks for the interest at our POC code. I appreciate your effort to improve the project.
Yes we accepting pulls!

Thanks

**Comment by rfdslabs** [comment]
> is this how you would use this?
> add_target 127.0.0.1 ciscoadmin ciscoadmin ftp://admin:1234@192.168.1.1/mimosa_logs
> 
> that's obviously all fake info for security reasons is that correct usage?

this is just for tests purpose.

**Comment by rfdslabs** [comment]
@jespinhara look here!

**Comment by devcoinfet** [comment]
No problem just finishing another project related to cisco stay tuned for that as well on my github I'm sure You guys will like it.  I will probably roll out the first copy of my improvements in the next few weeks for you to review and check if its correct i'm pretty sure it will be and works in the lab.  But like I had explained You guys built this for a cisco with tacas I changed that and allowed for other things as well.
